libs/wallet/src/utils.ts

```diff
@@ -1,16 +1,44 @@
 import { removeDecimal, toNanoSeconds } from '@vegaprotocol/utils';
+import * as Schema from '@vegaprotocol/types';
 import type { Market, Order } from '@vegaprotocol/types';
-import { OrderTimeInForce, OrderType, AccountType } from '@vegaprotocol/types';
+import { OrderTimeInForce, OrderType } from '@vegaprotocol/types';
 import BigNumber from 'bignumber.js';
 import { ethers } from 'ethers';
 import { sha3_256 } from 'js-sha3';
+import type { Transfer } from '@vegaprotocol/protos/dist/vega/commands/v1/Transfer';
 import type {
-  OrderAmendment,
-  OrderSubmission,
   Transaction,
-  Transfer,
+  OrderSubmission,
+  OrderAmendment,
 } from './connectors';
 import type { Exact } from 'type-fest';
+import { vega as vegaProtos } from '@vegaprotocol/protos';
+import type { Side } from '@vegaprotocol/protos/dist/vega/Side';
+import type { TimeInForce } from '@vegaprotocol/protos/dist/vega/Order/TimeInForce';
+import type { Type } from '@vegaprotocol/protos/dist/vega/Order/Type';
+
+export interface DealTicketOrderSubmission {
```

Comment:
These types shouldn't live here. They would be better in the deal ticket lib. The deal ticket should be required to conform to the interfaces provided by the wallet